src/qml/bitcoin_qml.qrc

```diff
@@ -34,6 +34,7 @@
         <file>pages/onboarding/OnboardingStorageAmount.qml</file>
         <file>pages/onboarding/OnboardingStorageLocation.qml</file>
         <file>pages/onboarding/OnboardingStrengthen.qml</file>
+        <file>pages/onboarding/OnboardingWizard.qml</file>
```

Comment:
We don't have `pages/onboarding/OnboardingWizard.qml`, do we?

Reply:
Good catch, missed a git add. It is included in the PR now.